app/routes/articles/components/ArticleList.tsx

```diff
@@ -91,13 +91,10 @@ const ArticleList = () => {
 
   const dispatch = useAppDispatch();
 
+  usePreparedEffect('fetchPopularTags', () => dispatch(fetchPopular()), []);
   usePreparedEffect(
-    'fetchArticlesOverview',
-    () =>
-      Promise.allSettled([
-        dispatch(fetchPopular()),
-        dispatch(fetchAll({ next: false, query })),
-      ]),
+    'fetchArticleList',
+    () => dispatch(fetchAll({ next: false, query })),
```

Comment:
Why are these separated?

Reply:
Wo don't need to re-fetch popular tags every time query changes